pormpt_test.go

```diff
@@ -0,0 +1,126 @@
+package main
+
+import (
+	"context"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestSetConfigPath(t *testing.T) {
+	var tests = []struct {
+		name     string
+		configs  []string       // input to the setConfigPath function
+		cr       ContentReader  // input to the setConfigPath function
+		prompt   string         // user interaction with the terminal
+		expected *InitActionCmd // expected command after user interaction
+	}{
+		{
+			name:     "no config files",
+			configs:  []string{},
+			expected: &InitActionCmd{},
+		},
+		{
+			name: "1 config file, dont use it",
+			cr: &mockContentReader{
+				content: `env "local" {}`,
+			},
+			configs: []string{"atlas.hcl"},
+			// arrow key down and then enter
+			prompt: "\x1b[B\n\n",
+			expected: &InitActionCmd{
+				ConfigPath: "",
+			},
+		},
+		{
+			name: "1 config file, use it",
+			cr: &mockContentReader{
+				content: `env "local" {
+                             dev = "postgres://localhost:5432/dev"
+						}`,
+			},
+			configs: []string{"atlas.hcl"},
+			prompt:  "\n",
+			expected: &InitActionCmd{
+				ConfigPath: "atlas.hcl",
+				ConfigEnv:  "local",
+				HasDevURL:  true,
+			},
+		},
+		{
+			name: "1 config file, witout dev url",
+			cr: &mockContentReader{
+				content: `env "local" {}`,
+			},
+			configs: []string{"atlas.hcl"},
+			prompt:  "\n",
+			expected: &InitActionCmd{
+				ConfigPath: "atlas.hcl",
+				ConfigEnv:  "local",
+			},
+		},
+		{
+			name: "2 config files, use second",
+			cr: &mockContentReader{
+				content: `env "local" {
+                             dev = "postgres://localhost:5432/dev"
+						}`,
+			},
+			configs: []string{"atlas.hcl", "atlas2.hcl"},
+			// arrow key down, enter, enter
+			prompt: "\x1b[B\n\n",
+			expected: &InitActionCmd{
+				ConfigPath: "atlas2.hcl",
+				ConfigEnv:  "local",
+				HasDevURL:  true,
+			},
+		},
+		{
+			name: " 2 config files, multiple envs, select first file and second env",
+			cr: &mockContentReader{
+				content: `env "local" {
+                             dev = "postgres://localhost:5432/dev"
+						}
+						env "prod" {
+                             dev = "postgres://localhost:5432/dev"
+						}`,
```

Comment:
```suggestion
				content: `
env "local" {
  dev = "postgres://localhost:5432/dev"
}
env "prod" {
  dev = "postgres://localhost:5432/dev"
}`,
```
make it easier to read